yamale/VERSION

```diff
@@ -1 +1 @@
-5.0.0
+5.1.0
```

Comment:
Please verify that this version is correct, because the description of [the commit](https://github.com/23andMe/Yamale/pull/243/commits/2339d78bb7d3faedde7febb3f49f631646816fa2) that added this change says "Prepare for **5.0.1** release".

Reply:
Thanks, was intending on 5.1.0, fixed.